ui/apps/platform/src/Containers/Compliance/Entity/Cluster.js

```diff
@@ -12,11 +12,12 @@ import ComplianceList from 'Containers/Compliance/List/List';
 import ComplianceByStandards from 'Containers/Compliance/widgets/ComplianceByStandards';
 import Loader from 'Components/Loader';
 import BackdropExporting from 'Components/PatternFly/BackdropExporting';
-import ResourceTabs from 'Components/ResourceTabs';
 import { entityPagePropTypes, entityPageDefaultProps } from 'constants/entityPageProps';
 import searchContext from 'Containers/searchContext';
 import isGQLLoading from 'utils/gqlLoading';
+
 import Header from './Header';
+import ResourceTabs from './ResourceTabs';
```

Comment:
I assume this was an intentional move, splitting off imports that are outside the current container from the ones that are not?

Reply:
Yes, good question. After scavenger hunt for recent deletions, whenever I investigate to see scope of incremental changes to a component file, and find that only one container imports it, I move into the container.